libeos-payg/manager.c

```diff
@@ -954,15 +974,35 @@ epg_manager_init_async (GAsyncInitable      *initable,
 
   EpgManager *self = EPG_MANAGER (initable);
   g_autoptr(GTask) task = g_task_new (self, cancellable, callback, user_data);
+  g_autoptr(GFile) expiry_time_file = get_expiry_time_file (self);
+
   g_task_set_source_tag (task, epg_manager_init_async);
   g_task_set_priority (task, priority);
   epg_multi_task_attach (task, 4);
```

Comment:
In the case where you're making 1 extra async call, this will need incrementing.